README.md

```diff
@@ -28,14 +28,26 @@ which is located in `tests/conftest.py`.
 # Setup
 Create and activate your conda environment with python3.9:
 ```commandline
-conda create -y -n substrait_consumer_testing -c conda-forge python=3.9
+conda create -y -n substrait_consumer_testing -c conda-forge python=3.9 openjdk
 conda activate substrait_consumer_testing
 ```
+*Note: Java is used by Jpype to access the Isthmus producer.  
+JPype has been tested with Java versions from Java 1.7 to Java 13.
+However, it should work with all versions of Java.*
 
 Install requirements from the top level directory:
 ```commandline
 pip install -r requirements.txt
 ```
+
+Get the java dependencies needed by the Isthmus Substrait producer:
+```commandline
+git clone https://github.com/substrait-io/substrait-java.git
+export SUBSTRAIT_JAVA_HOME=$(pwd)/substrait-java
+cd consumer-testing
+make
```

Comment:
What does the error look like if the user hasn't run `make`?  Is it obvious that running `make` is what they need to do?

Similarly, if a user runs `make` and doesn't set `SUBSTRAIT_JAVA_HOME` I don't think the error will be obvious.  Can you test for the `${SUBSTRAIT_JAVA_HOME}/isthmus` directory and give a nice error about setting the environment variable if it doesn't exist?

Reply:
If the user doesn't run the script that builds/copies isthmus, but tries to run the tests still, they'll see an error like this:
```
E   TypeError: Class io.substrait.isthmus.SqlToSubstrait is not found
```